Dockerfile

```diff
@@ -18,6 +17,9 @@ LABEL org.label-schema.build-date=$BUILD_DATE \
       org.label-schema.vcs-url="https://github.com/kbase/Shock.git" \
       org.label-schema.vcs-ref=$VCS_REF \
       org.label-schema.schema-version="1.0.0-rc1" \
-      us.kbase.vcs-branch=$BRANCH
+      us.kbase.vcs-branch=$BRANCH \
+      maintainer="Steve Chan sychan@lbl.gov"
 
-ENTRYPOINT [ "/kb/deployment/bin/entrypoint.sh" ]
+ENTRYPOINT [ "/kb/deployment/bin/dockerize"]
```

Comment:
So dockerize completely replaces the entrypoint script, and everything that it calls is put into docker-compose.yml?

Reply:
Yep! The entrypoint script and it's dependencies ( python, wget ) are all gone, trimming 30M off the image size. Even after extending dockerize to pull data from remote INI files, it only expanded the dockerize binary by a couple of megs.